packages/block-editor/src/components/block-toolbar/utils.js

```diff
@@ -153,6 +158,8 @@ export function useShowMoversGestures( {
 	};
 }
 
+let requestAnimationFrameId;
```

Comment:
Thank you for improving this! 🙏

Reply:
It could use some explanation in the code. There can be multiple instances of the block toolbar. I read that it's related to the focus that can be in one place so it's perfectly fine. Anyway, it's something that might be unclear for the reader at first glance.